t_cam_to_body:
- [0.111125]
- [0.005]
- [-0.03174]
- [0]
dt: 0.05
occupancy_map:
- [1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1]
- [0, 0, 1, 1, 1, 0, 0, 0, 0, 0, 1, 1, 1, 0, 0, 0, 0, 0, 0, 1]
- [0, 0, 1, 1, 1, 0, 0, 0, 0, 0, 1, 1, 1, 0, 0, 0, 0, 0, 0, 1]
- [0, 0, 1, 1, 1, 0, 1, 1, 1, 0, 1, 1, 1, 0, 1, 1, 1, 0, 0, 1]
- [0, 0, 1, 1, 1, 0, 1, 1, 1, 0, 1, 1, 1, 0, 1, 1, 1, 0, 0, 1]
- [0, 0, 0, 0, 0, 0, 1, 1, 1, 0, 0, 0, 0, 0, 1, 1, 1, 0, 0, 1]
- [0, 0, 0, 0, 0, 0, 1, 1, 1, 0, 0, 0, 0, 0, 1, 1, 1, 0, 0, 1]
- [1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1]
max_omega: 1.6
max_vel: 0.3
pos_init:
- [1.2]
- [0.05]
- [3.14159]
pos_goal:
- [0.]
- [0.075]
- [3.14159]
world_map:
- [0.3, 0.05, 3.14159., 0.0]
- [0.6, 0.35, 3.14159., 1.0]
- [0.9, 0.05, 3.14159, 2.0]
- [1.2, 0.35, 3.14159, 3.0]
- [-0.30, 0.05, 3.14159, 4.0]
- [0., 0.35, 3.14159, 5.0]
x_spacing: 0.08
y_spacing: 0.05
